variables.tf

```diff
@@ -5,12 +5,29 @@ variable "api_key" {
   sensitive   = true
 }
 
+variable "api_key_name" {
```

Comment:
I'm a bit confused why api_key is not a required variable, now it makes sense but it did not make sense before this PR. 

Could you check the basic example? Should we not provide an api key here?

Could you also make an example where you let the module create the api key?

Reply:
I've read the module again, so the module does 2 things. 

- Setup the integration
- (optional) setup log forwarding (which requires an api key). This now has 2 flavours, provide our own key or let the key be created.

Could you alter the basic example to only setup integration. and add log forwarding examples? 1 with providing your own key, and 1 where the module creates the key